Jenkinsfile

```
@@ -228,7 +228,15 @@ pipeline {
 										if (isUnix()){
 											sh '''
 												mkdir libs
-												sh build.sh install
+												if [[ ${PLATFORM} == gtk.linux.aarch64 ]]; then
+													sh build.sh -gtk3 install
+												elif [[ ${PLATFORM} == gtk.linux.ppc64le ]]; then
+													sh build.sh -gtk3 install
+												elif [[ ${PLATFORM} == gtk.linux.riscv64 ]]; then
+													sh build.sh -gtk3 install
+												else
+													sh build.sh install 
+												fi
```

If the script becomes a Groovy String-template (by using three double-quotes instead of three single-quotes), this part can be much shorter and much more expressive
```suggestion
   sh build.sh ${ os == 'linux' && arch != 'x86_64' ? '-gtk3' : '' } install
```

---

As the goal is to successively build for other architectures as well I choose a "longer" form so it is more clear what to remove if one enable a certain architecture. Hopefully the whole block can even be removed again soon.